packages/calling/src/CallHistory/CallHistory.ts

```diff
@@ -139,6 +139,14 @@ export class CallHistory extends Eventing<CallHistoryEventTypes> implements ICal
         }
       }
     );
+    this.sdkConnector.registerListener<CallSessionEvent>(
+      MOBIUS_EVENT_KEYS.LOCUS_CALL_SESSION_EVENT,
+      async (event?: CallSessionEvent) => {
```

Comment:
Seems like this is just a duplicate of the code on lines 136-139? Could we optimize?

Reply:
done